Ui dim

heartjigi opened the pull request:
make dimension flexible on each device.
Almost "fixed value of dim" change dynamic

Commit:
make new branch

Commit:
change dim on userList

Commit:
Merge branch 'master' of https://github.com/sv-bootcamp/bridgeme-client into UI_dim

Commit:
change dim on Sign Up

Commit:
Merge branch 'master' of https://github.com/sv-bootcamp/bridgeme-client into UI_dim

Commit:
edit ui on util

Commit:
Merge branch 'master' of https://github.com/sv-bootcamp/bridgeme-client into UI_dim

heartjigi committed:
edit on userprofile

heartjigi committed:
edit onboarding

heartjigi committed:
edit Login

heartjigi committed:
edit Chat

heartjigi committed:
edit tab icons and chat dim

heartjigi committed:
edit right text style on bar

heartjigi committed:
edit activity

heartjigi committed:
pull from master

heartjigi committed:
fix save button on tap

heartjigi committed:
pull from master

heartjigi committed:
modify onboarding

heartjigi committed:
onboarding

heartjigi committed:
fix onboarding

heartjigi committed:
Merge branch 'master' of https://github.com/sv-bootcamp/bridgeme-client into UI_dim

heartjigi committed:
save

heartjigi committed:
save before pull

heartjigi committed:
Merge branch 'master' of https://github.com/sv-bootcamp/bridgeme-client into UI_dim

# Conflicts:
#	ios/Bridgeme.xcodeproj/project.pbxproj

heartjigi committed:
1. Navi bar style change
2. Some font apply on MyPage
3. Change tab bar style on Activity
4. Change right font style on sign up components
5. Change card view dim by dynamic
6. Change onboarding title font

heartjigi committed:
change meaningless

heartjigi committed:
no message

heartjigi committed:
pull from master

SarahhhC commented on src/components/OnBoarding/OnBoardingPage.js:
```diff
@@ -69,32 +79,64 @@ class OnBoardingPage extends Component {
     );
   }
 
+  controlScroll(e, state) {
```
What is `e`?

heartjigi replied:
e is event,
This is original case of onMomentumScrollEnd for Scrollview,
for get state, e is needed maybe..

SarahhhC replied:
Can't it be `event` instead `e`?

heartjigi replied:
solved

SarahhhC commented on src/components/Shared/Dimensions.js:
```diff
@@ -0,0 +1,16 @@
+  height: 667,
+};
+export const dimensions = {
+  width: Dimensions.get('window').width,
```
Why don't you use `Dimensions.get('window').width` or other things as variable?

heartjigi replied:
Sry I can't understand your intend,
Is that not variable?

SarahhhC commented on src/utils/UserUtil.js:
```diff
@@ -13,7 +13,8 @@ class UserUtil {
   // Sign In with Facebook.
   signInWithFacebook(callback) {
-    this.signOut();
+    console.log('signInWithFacebook');
```
delete console.

heartjigi replied:
Oh... eagle eye.. Thanks

SarahhhC reviewed (commented):
Please check comments!

heartjigi committed:
delete console.log

heartjigi committed:
apply comments

heartjigi committed:
Merge branch 'master' into UI_dim

heartjigi committed:
Merge branch 'master' of https://github.com/sv-bootcamp/bridgeme-client into UI_dim

# Conflicts:
#	src/components/Activity/NewRequestsRow.js
#	src/components/userProfile/RequestPage.js

heartjigi committed:
pull from master

heartjigi committed:
edit margin of main

heartjigi committed:
Merge branch 'master' of https://github.com/sv-bootcamp/bridgeme-client into UI_dim

# Conflicts:
#	src/components/AppProps.js
#	src/components/Main.js
#	src/components/SignUp/ExpertInfo.js
#	src/components/UserList/Row.js
#	src/utils/dropdown/option.js

heartjigi committed:
apply filter new dim,
erase useless statusbar

heartjigi committed:
Merge branch 'master' of https://github.com/sv-bootcamp/bridgeme-client into UI_dim

# Conflicts:
#	src/components/SignUp/EduForm.js
#	src/components/SignUp/EduFormIOS.js
#	src/components/SignUp/Styles.js
#	src/components/SignUp/WorkForm.js

heartjigi committed:
fix lint

heartjigi committed:
rebase chat dim

heartjigi committed:
Merge branch 'master' of https://github.com/sv-bootcamp/bridgeme-client into UI_dim

# Conflicts:
#	src/components/AppProps.js
#	src/components/Main.js

heartjigi committed:
Insert indicator image on edit profile page
Edit filter button margin
Fix some error on main js ,on refresh right image source '' -> null

ifndefdeadmau5 commented:
LGTM Very good

[![Approved with PullApprove](https://img.shields.io/badge/code--review-approved-brightgreen.svg)](https://pullapprove.com/sv-bootcamp/bridgeme-client/pull-request/139/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=sv-bootcamp/bridgeme-client)
